LICENSE

```diff
@@ -1,2 +1,2 @@
-Copyright 2019 Google LLC.
-SPDX-License-Identifier: Apache-2.0
+Copyright 2019 Google LLC.,JCS-Enterprises-LLC (c)2021
```

Comment:
[privacy-model-7cba0c4e7d080ebb9779e7d7f60b9a57f219fd18.zip](https://github.com/michaelkleber/privacy-model/files/7094600/privacy-model-7cba0c4e7d080ebb9779e7d7f60b9a57f219fd18.zip)

Reply:
Why would this PR make sense?